node/src/effect/announcements.rs

```diff
@@ -44,6 +48,49 @@ pub(crate) enum ControlAnnouncement {
         /// Error message.
         msg: String,
     },
+    // An external event queue dump has been requested.
+    QueueDumpRequest {
```

Comment:
Why is the other request for diagonostic console not part of this `ControlAnnouncement` enum?

Reply:
Because `ControlAnnouncement` (which should actually be named `ControlEvent` now, tbh) groups all events that are handled outside of the reactor by the runtime directly.

The other request is for data from consensus and requires no direct interventions from the `Runner`, as opposed to this one.

Reply:
Ah, of course. Makes a lot of sense. Thanks.